2020-04-08-devnet-upgraded.md

```diff
@@ -0,0 +1,21 @@
+# Devnet has been upgraded to rippled 1.6.0-b1
+
+Ripple operates a test network called "Devnet" as a public service. To create a Devnet test account with test XRP, or to access Ripple's Devnet servers, see https://xrpl.org/xrp-testnet-faucet.html .
+
+As of April 8, 2020, the Devnet validators have been upgraded to rippled version 1.6.0-b1.
+
+The Devnet validators currently operate with the following veto_amendments stanza:
+
+[veto_amendments]
+C6970A8B603D8778783B61C0D445C23D1633CCFAEF0D43E7DBCD1521D34BD7C3 SHAMapV2
+C1B8D934087225F509BEB5A8EC24447854713EE447D277F69545ABFA0E0FD490 Tickets
+9178256A980A86CF3D70D0260A7DA6402AAFE43632FDBCB88037978404188871 OwnerPaysFee
+86E83A7D2ECE3AD5FA87AB2195AE015C950469ABF0B72EAACED318F74886AE90 CryptoConditionsSuite
+
+Amendments not listed above should be available for testing on Devnet in 2 weeks or less. The full list of Known Amendments: https://xrpl.org/known-amendments.html
```

Comment:
There's a new amendment that's not on that page yet. Should I fix it? Also, maybe we should include that in this announcement?

Reply:
Yes, I think it would be good to add all new amendments to that page. Yes, I'll try adding the amendment to this announcement as well.